.index.json

```diff
@@ -424,6 +448,41 @@
    "author": "crowdsecurity",
    "labels": null
   },
+  "crowdsecurity/postfix-logs": {
+   "path": "parsers/s01-parse/crowdsecurity/postfix-logs.yaml",
+   "stage": "s01-parse",
+   "version": "0.6",
+   "versions": {
+    "0.1": {
```

Comment:
Same here why there is 6 versions ? and we are only in the first iteration of the scenario